pkg/apis/flyteworkflow/v1alpha1/node_status.go

```diff
@@ -403,6 +403,9 @@ func (in *NodeStatus) UpdatePhase(p NodePhase, occurredAt metav1.Time, reason st
 		if in.StartedAt == nil {
 			in.StartedAt = &n
 		}
+		if in.LastAttemptStartedAt == nil {
```

Comment:
This means it'll only be set once, right? shouldn't this be set when we update the status to running again?

Reply:
Correcto. I think its being reset after we retry. If not then we should do that but not remove null check.

Reply:
https://github.com/lyft/flytepropeller/blob/master/pkg/controller/nodes/executor.go#L392